pkgs/servers/sql/postgresql/ext/sqlite_fdw.nix

```
@@ -0,0 +1,40 @@
+{
+  lib,
+  fetchFromGitHub,
+  stdenv,
+  sqlite,
+  postgresql,
+}:
+
+stdenv.mkDerivation {
+  pname = "sqlite_fdw";
+  version = "2.4.0";
+
+  src = fetchFromGitHub {
+    owner = "pgspider";
+    repo = "sqlite_fdw";
+    rev = "v2.4.0";
+    hash = "sha256-u51rcKUH2nZyZbI2g3crzHt5jiacbTq4xmfP3JgqnnM=";
+  };
+
+  buildInputs = [
+    sqlite
+    postgresql
+  ];
+
+  makeFlags = [ "USE_PGXS=1" ];
+
+  installPhase = ''
+    mkdir -p $out/lib $out/share/postgresql/extension
+    mv *.so $out/lib
+    mv *.sql *.control $out/share/postgresql/extension
+  '';
```

```suggestion
  installPhase = ''
    runHook preInstall
    mkdir -p $out/lib $out/share/postgresql/extension
    mv *.so $out/lib
    mv *.sql *.control $out/share/postgresql/extension
    runHook postInstall
  '';
```
I think you need to use *.dylib or something on Darwin.